src/GafferScene/ObjectSource.cpp

```diff
@@ -310,18 +315,14 @@ GafferScene::ConstPathMatcherDataPtr ObjectSource::computeSet( const IECore::Int
 	}
 }
 
-IECore::InternedString ObjectSource::standardSetName() const
+IECore::ConstInternedStringVectorDataPtr ObjectSource::computeStandardSetNames( const Gaffer::Context *context ) const
 {
-	return g_emptyString;
+	IECore::InternedStringVectorDataPtr result = new IECore::InternedStringVectorData();
+	return result;
 }
 
 bool ObjectSource::setNameValid( const IECore::InternedString &setName ) const
 {
-	if( setName != g_emptyString && setName == standardSetName() )
```

Comment:
I suspect this is the cause of the test failures. Just as before we returned true if setName was equal to `standardSetName()`, we will now need to return true if setName is in `standardSetNames()`.